Completed schema for new applicant user

jinsim95 opened the pull request.

mattfan00 commented:
Hey Jin, sorry to snoop around. This is looking good! I just have a couple of suggestions :)

I believe that the details field should be an object instead of an array (it should be ok to just make it an empty object, so you don't have to declare fields within it)

Additionally I was thinking we could add a password field so that we can start registering and logging in users. 

Let me know what you think!

jinsim95 commented:
@mattfan00 Hey, no problem and thanks for the comment! I've fixed the details to be just an empty object and I've also added 'password' field. Please take a look at it and lmk if it looks okay to use.

mattfan00 commented:
looks good, ty!
